shell/app/common/constants.ts

```diff
@@ -12,7 +12,9 @@
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 export const WORKSPACE_LIST = ['DEV', 'TEST', 'STAGING', 'PROD'];
+export const ROOT_DOMAIN = 'erda.cloud';
```

Comment:
is there any impact if deploy as private domain. e.g. erda.ubras.cn ?

Reply:
The first level part of path will always be the org name, it does not matter.